meta-luneos/recipes-connectivity/voicecall/voicecall/0001-Qt6-migration-patch.patch

```diff
@@ -15,6 +15,19 @@ Upstream-Status: Pending
  src/basicringtonenotificationprovider.cpp     | 10 +++++---
  7 files changed, 28 insertions(+), 19 deletions(-)
 
+diff --git a/src/src.pro b/src/src.pro
+index d50a6b1..2052b2a 100644
+--- a/src/src.pro
++++ b/src/src.pro
+@@ -5,7 +5,7 @@ CONFIG += link_pkgconfig
+ 
+ INCLUDEPATH += ../lib/src
+ 
+-DEFINES += VOICECALL_PLUGIN_DIRECTORY=\"\\\"$$[QT_INSTALL_LIBS]/voicecall/plugins\\\"\"
++DEFINES += VOICECALL_PLUGIN_DIRECTORY=\"\\\"/usr/lib/voicecall/plugins\\\"\"
```

Comment:
@tofee @shr-project Shouldn't we be using QT6_INSTALL_LIBDIR here? https://code.qt.io/cgit/yocto/meta-qt6.git/plain/classes/qt6-paths.bbclass

Reply:
I tried that, but it resulted in an empty path... but maybe I didn't pass it correctly to qmake, I'll give it another try.

Reply:
Also even qt5 used to use /usr/lib/qt5 as QT_INSTALL_LIBS before switching back to just /usr/lib, so it might not matter for voicecall, but hardcoding "/usr/lib" might not work in other cases (as well as with multilib).

Reply:
New version now uses QT6_INSTALL_LIBDIR from meta-qt6